tests/data/lbg/gedi/GEDI02_B_2019228092851_O03828_T04236_02_001_01/__GEDI02_B_2019228092851_O03828_T04236_02_001_01_gedi_l2b_cover_z.yaml

```diff
@@ -0,0 +1,120 @@
+id: 28de4ad6-e0c7-4443-b0c9-1f6e99d86086
+$schema: https://schemas.opendatacube.org/dataset
+product:
+  name: gedi_l2b_cover_z
+crs: epsg:4326
+grids:
+  default:
+    shape:
+      - 415
+      - 548
+    transform:
+      - 0.00027778
+      - 0.0
+      - 149.04036673536626
+      - 0.0
+      - -0.00027778
+      - -35.30377268593081
+      - 0.0
+      - 0.0
+      - 1.0
+measurements:
+  "5":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "10":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "15":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "20":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "25":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "30":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "35":
+    layer: array
+    path: ./GEDI02_B_2019228092851_O03828_T04236_02_001_01_cover_z.xarray_3d
+  "40":
+    layer: array
```

Comment:
I'm confused about this, I thought there would be only one measurement per ND dataset, and product definition would cover how to interpret extra dimension. But this looks like you have to list all, potentially hundreds, of sub-bands. We could that before.

Reply:
Will do.

Most of these were for backwards compatibility for loading a 3D measurement as though they are a list of 2D measurements. A 3D measurement is the main measurement, and a bunch of 2D measurement are a auto-generated from the 3D measurement. The 2D auto-generated components can go.

It means you can only do a
 - `dc.load(...measurements=['cover_z'], z=(5,10)...)`
    - Gives you a 3D result
 - and not a 
 - `dc.load(...measurements=['5', '10']')`
    - Give you a list of 2D results.

I'll remove the 2D auto-generated stuff as well as making the dataset yaml a single 3D measurement with one uri.

Reply:
Done, Please let me know if there are any other required changes.